cfme/automate/service_dialogs.py

```diff
@@ -1,84 +1,109 @@
-# -*- coding: utf-8 -*-
-import functools
-
-from navmazing import NavigateToSibling, NavigateToAttribute
-
 from cfme.fixtures import pytest_selenium as sel
-from cfme.web_ui import toolbar as tb
-from cfme.web_ui import AngularSelect, Form, Select, SplitTable, accordion,\
-    fill, flash, form_buttons, Table, Tree, Input, Region, BootstrapTreeview
-from utils import version
-from utils.appliance import Navigatable
-from utils.appliance.implementations.ui import navigator, navigate_to, CFMENavigateStep
-from utils.pretty import Pretty
+from widgetastic.widget import Text, Checkbox
+from widgetastic_patternfly import Button, Input, BootstrapSelect, Dropdown
+from widgetastic_manageiq import ManageIQTree, Table
+from navmazing import NavigateToAttribute, NavigateToSibling
 from utils.update import Updateable
+from utils.pretty import Pretty
+from utils.appliance import Navigatable
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from utils import version
+
+
+from . import AutomateCustomizationView
+
+
+class LabelForm(AutomateCustomizationView):
+    title = Text('#explorer_title_text')
+
+    label = Input(name='label')
+    description = Input(name="description")
+
+    submit_button = Checkbox(name='chkbx_submit')
+    cancel_button = Checkbox(name='chkbx_cancel')
+
+    plus_btn = Dropdown('Add')
+
+    tab_label = Input(name='tab_label')
+    tab_desc = Input(name="tab_description")
+
+    box_label = Input(name='group_label')
+    box_desc = Input(name="group_description")
+
+    ele_label = Input(name='field_label')
+    ele_name = Input(name="field_name")
+    ele_desc = Input(name="field_description")
+    choose_type = BootstrapSelect('field_typ')
+    default_text_box = Input(name='field_default_value')
+    default_value = Checkbox(name='field_default_value')
+    field_required = Checkbox(name='field_required')
+    field_past_dates = Checkbox(name='field_past_dates')
+    field_entry_point = Input(name='field_entry_point')
+    field_show_refresh_button = Input(name='field_show_refresh_button')
+    entry_value = Input(name='entry[value]')
+    entry_description = Input(name='entry[description]')
+    add_entry_button = Input(name='accept')
+    field_category = BootstrapSelect('field_category')
+    text_area = Input(name='field_default_value')
+    dynamic_chkbox = Input(name='field_dynamic')
+    element_tree = ManageIQTree('dialog_edit_treebox')
```

Comment:
Which tree is this one? The one on the left side? I believe it should be at least visually separated.